examples/torch_onnx/requirements.txt

```diff
@@ -0,0 +1,7 @@
+datasets>=2.14.4
+onnx
+onnx_graphsurgeon
+packaging
```

Comment:
these 3 dependencies should be installed as part of `nvidia-modelopt[onnx]` hence dont need to be mentioned separately here